libvmaf/src/feature/feature_collector.h

```diff
@@ -42,15 +43,32 @@ typedef struct {
     unsigned cnt, capacity;
 } AggregateVector;
 
+typedef struct {
+        void (*callback)(void *, const char * ,double);
+        void *data;
+        size_t data_sz;
+} VmafMetadata;
+
+typedef struct VmafPredictModel VmafPredictModel;
```

Comment:
Nit: you can avoid the duplicated typedef by using `struct VmafPredictModel *next` inside the `VmafPredictModel` definition (usages outside of this definition can remain the same)

Reply:
Thanks for the feedback! You are right, fixed in local.